osbenchmark/telemetry.py

```diff
@@ -614,6 +614,7 @@ class NodeStats(TelemetryDevice):
     warning = """You have enabled the node-stats telemetry device with OpenSearch < 1.1.0. Requests to the
           _nodes/stats OpenSearch endpoint trigger additional refreshes and WILL SKEW results.
     """
+    opensearch_distribution = "opensearch"
```

Comment:
Might rename this as `opensearch_distribution_name` to be consistent with the variable names below.